js/magazine-issue.js

```diff
@@ -0,0 +1,15 @@
+(function ($) {
+
+	function toggle_excerpt() {
+		$('.halves:not(.secondary-articles)').on('click', '.home-headline-head-wrapper', function (e) {
+			if ( $(window).width() < 792 ) {
+				e.preventDefault();
+				e.stopImmediatePropagation();
+				$(this).toggleClass('open');
+			}
+		});
+	}
+
+	$(window).on('load resize', toggle_excerpt);
```

Comment:
This will add the same click event every time the browser is resized. It's not really that big of a deal to most visitors, though it could cause confusion in some cases if the `open` class starts getting double-toggled. I'm not entirely sure whether that would happen or not.

One approach is probably to only add the click action on load. Then it will fire on all browser sizes, but only take action if the width is less than 792.

Another would be to add/remove the click action as the browser resizes, but that's probably going too far.

So I guess, my recommendation would be to drop the `resize`. :)